renderer-macos/lib/components/Button.re

```diff
@@ -21,18 +21,15 @@ let make =
         style
         |> List.iter(attr =>
              switch (attr) {
-             | `Color(_) => ()
-             | `Background(({r, g, b, a}: Color.t)) =>
-               BriskView.setBackgroundColor(view, r, g, b, a)
-             | `Border(({width, color}: Border.t)) =>
-               if (!isUndefined(width)) {
-                 BriskView.setBorderWidth(view, width);
-               };
-               let {r, g, b, a}: Color.t = color;
-               BriskView.setBorderColor(view, r, g, b, a);
+             | `Background(_) =>
+               BriskButton.setIsBordered(view, false);
```

Comment:
You cannot change background color of `NSButton` unless it's not bordered (unless we go the route of redefining `draw`).

Reply:
that's terrible. We need a runtime exception for that or model this in types.

Reply:
I agree. We could keep `Button` as a custom-styled button, and add some other component like `ButtonMacos` for an Apple-styled component with bezels and whatnot.

This will result in dropping the `` `Background`` prop in `Button` style.